### DISCUSSION TITLE
how safe is the wrappers? could i login to my accounts with it?

### DISCUSSION THREAD

**Discussion by payrim** [mdSilo/mdSilo-app#487] [Q&A] [answered]
was just wondering, my RSS is one place, bookmarks another place, nativified apps in somewhere else and notes too, and another program for week scheduling (my mind can't take this!!). if i login to my accounts with mdSilo is it safe?

also it would be, idk if possible, support for a shell or custom commands in the future. would like to integrate some of my python CLI tracker scripts too!

**Comment by danloh** [comment, marked as answer]
It is a local app, the wrapper is just a wrapper for the web app and you can inject some Javascripts at your risk(like a extension for the web browser),  no data transfer or store, no server communication. ALL the data stored in your device.
